lib/test-data.json

```diff
@@ -88,8 +88,8 @@
   ],
   "edges": [
     {
-      "source": "a",
-      "target": "g",
+      "source": "0+a",
```

Comment:
If the edge is scoped, why does it refer to the global ID of its source and target?